test/functional/mempool_rebroadcast.py

```diff
@@ -0,0 +1,230 @@
+#!/usr/bin/env python3
+# Copyright (c) 2009-2019 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test mempool rebroadcast logic.
+
+"""
+
+from collections import defaultdict
+from test_framework.mininode import P2PInterface, mininode_lock
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import (
+        assert_equal,
+        assert_greater_than,
+        wait_until,
+        disconnect_nodes,
+        connect_nodes,
+        gen_return_txouts,
+        create_confirmed_utxos,
+        create_lots_of_big_transactions,
+)
+import time
+
+# Constant from txmempool.h
+MAX_REBROADCAST_WEIGHT = 3000000
+
+class P2PStoreTxInvs(P2PInterface):
```

Comment:
this is a temporary parking spot for this code. All three tests use it, so I'd like to pull it out into some shared utility, but haven't figured out where makes sense yet. Any suggestions?

Reply:
It doesn't depend on any outside modules except `mininode`, so it could be moved there or to a new module.